mmocr/utils/ocr.py

```diff
@@ -350,7 +361,13 @@ def __init__(self,
                     ' with text detection and recognition algorithms.')
 
         self.detect_model = None
-        if self.td:
+        if self.td and self.td == 'Tesseract_det':
```

Comment:
```python
        if self.td and self.td == 'Tesseract':
            ...
            self.detection_model = 'Tesseract_det'
```
We don't need to expose the lengthy name to users, and it's better to distinguish the model type internally. (Same for the recognizer)